### PR TITLE
Added CRUD repository

### PR DESCRIPTION
**Pull request by NiazBikbaev**

### CODE REVIEW

File: src/main/java/com/epam/magazinestore/repository/AccountRepository.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+import com.epam.magazinestore.entity.Account;
+import org.springframework.data.repository.CrudRepository;
+
+/**
+ * Performs basic CRUD operations
+ *
+ * @author Niyaz Bikbaev
+ */
+public interface AccountRepository extends CrudRepository<Account, Long> {
```
Comment (Jebsuz) [inline src/main/java/com/epam/magazinestore/repository/AccountRepository.java]: Why did you choose `CrudRepository` over `JpaRepository`?
Did you read about different `Repository`'s that Spring Data provides?

Comment (Jebsuz) [inline src/main/java/com/epam/magazinestore/repository/AccountRepository.java]: The stereotype `@Repository` annotation wouldn't hurt

Comment (NiazBikbaev) [reply src/main/java/com/epam/magazinestore/repository/AccountRepository.java]: By default spring boot searches repositories from the package containing main configuration class, thus @Repository annotation not needed

Comment (NiazBikbaev) [reply src/main/java/com/epam/magazinestore/repository/AccountRepository.java]: JpaRepository extends PagingAndSortingRepository which in turn extends CrudRepository. I chose this interface to make sure that the basic database operations works. In the future, we will use JpaRepository

Comment (Jebsuz) [reply src/main/java/com/epam/magazinestore/repository/AccountRepository.java]: Yea, i know, but one could not know this and thus some explicit annotation wouldn't hurt

Comment (Jebsuz) [reply src/main/java/com/epam/magazinestore/repository/AccountRepository.java]: I suggest you will start using more advanced interface right away, it has more handy methods for your needs, so why bother using basic `CrudRepository` if you already have a plans to change it?

**Pull request merged**